include/test/battle.h

```diff
@@ -741,6 +769,7 @@ struct moveWithPP {
 #define MovesWithPP(movewithpp1, ...) MovesWithPP_(__LINE__, (struct moveWithPP[MAX_MON_MOVES]) {movewithpp1, __VA_ARGS__})
 #define Friendship(friendship) Friendship_(__LINE__, friendship)
 #define Status1(status1) Status1_(__LINE__, status1)
+#define OtName(otName) do {static const u8 otName_[] = _(otName); OtName_(__LINE__, otName_);} while (0)
```

Comment:
```suggestion
#define OTName(otName) do {static const u8 otName_[] = _(otName); OTName_(__LINE__, otName_);} while (0)
```
(e.g. `MonOTNameNotPlayer`, `PrintEggOTName`)